.github/workflows/ci-wordpress.yaml

```diff
@@ -0,0 +1,98 @@
+name: CI-Wordpress
+
+# ----------------------------------------------------------------------------------------------------------
+# CI Pipeline for deploy wordpress to shared hosting by ansistrano ansible role
+#
+# Original Author: Rostislav Udaltsov
+# Created: 09/19/2023
+# ----------------------------------------------------------------------------------------------------------
+
+on:
+  workflow_call:
+    inputs:
+      # Build configuration
+      server_host:
+        required: true
+        type: string
+        description: 'The address of the server'
+      environment:
+        required: true
+        type: string
+        description: 'The environment'
+      playbook_path:
+        required: true
+        type: string
+        description: 'The path to the file relative to the repository'
+      deploy_path:
+        required: true
+        type: string
+        description: 'The path to deploy on the production server'
+      deploy_repo:
+        required: true
+        type: string
+        description: 'The repository for deploy'
+      deploy_branch:
+        required: true
+        type: string
+        description: 'The branche for deploy'
+      python_version:
+        required: true
+        type: number
+        description: 'The version of python'
+      runner_image:
+        required: true
+        type: string
+        description: 'The image of the runner OS on which workflow is running'
```

Comment:
This is wrong, see the next comment

Reply:
https://github.com/saritasa-nest/saritasa-github-actions/pull/6#discussion_r1331470809

Reply:
fix